easybuild/easyconfigs/f/FreeSurfer/FreeSurfer-7.3.2-centos8_x86_64.eb

```diff
@@ -0,0 +1,33 @@
+name = 'FreeSurfer'
+version = '7.3.2'
+versionsuffix = '-centos8_x86_64'
+
+homepage = 'https://surfer.nmr.mgh.harvard.edu/'
+description = """FreeSurfer is a set of tools for analysis and visualization 
+of structural and functional brain imaging data.  FreeSurfer contains a fully 
+automatic structural imaging stream for processing cross sectional and 
+longitudinal data."""
+
+toolchain = SYSTEM
+
+source_urls = [
+    'https://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/%(version)s/',
+    'https://surfer.nmr.mgh.harvard.edu/fswiki/MatlabRuntime?action=AttachFile&do=get&target='
```

Comment:
@maxim-masterov could you check this part, please? It failed to download in my first test build attempt.

Reply:
@verdurin Not sure why it fails, I've just re-installed it from this pr (removed old sources prior to it) and it worked

Reply:
Download works fine for me

Reply:
Yes, I retried and it was fine.